pkg/flavor/kubernetes/nfs.go

```diff
@@ -396,6 +406,23 @@ func (flavor *Flavor) GetNFSVolumeID(volumeID string) (string, error) {
 	return pv.Spec.PersistentVolumeSource.CSI.VolumeHandle, nil
 }
 
+func (flavor *Flavor) getNFSHostDomain() (string, error) {
+	log.Tracef(">>>>> getNFSHostDomain")
+	defer log.Tracef("<<<<< getNFSHostDomain")
+
+	// obtain an array of  {hostname, domainname}
+	chapiDriver := &chapi.LinuxDriver{}
```

Comment:
I meant passing in the reference created in the driver?  Why do we want to construct this in each function?

Reply:
```		driver.flavor.chapiDriver = driver.chapiDriver``` this way so that we can reference using flavor?

Reply:
Ya.  Then you just have a chapiDriver reference in flavor that was initialized in driver_<ostype>.go.

Reply:
understood..much cleaner..fixed it.